### PR TITLE
MOVE-142: Logging SR lookup

### PR DESCRIPTION
**Pull request by martinwam**

### CODE REVIEW

File: serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/logging/SRMarkerFactory.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package no.difi.meldingsutveksling.serviceregistry.logging;
+
+import net.logstash.logback.marker.LogstashMarker;
+import net.logstash.logback.marker.Markers;
+
+public class SRMarkerFactory {
```
Comment (nuth) [inline serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/logging/SRMarkerFactory.java]: MAJOR Add a private constructor to hide the implicit public one. [rule](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1118)

**Comment by nuth** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/logging/SRMarkerFactory.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    private SRMarkerFactory() {}
+
+    private static final String REMOTE_IP = "remote_ip";
```
Comment (nuth) [inline serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/logging/SRMarkerFactory.java]: MINOR Move this variable to comply with Java Code Conventions. [rule](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1213)

File: serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/logging/SRMarkerFactory.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    private static final String CLIENT_ID = "client_id";
```
Comment (nuth) [inline serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/logging/SRMarkerFactory.java]: MINOR Move this variable to comply with Java Code Conventions. [rule](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1213)

File: serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/logging/SRMarkerFactory.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    private static final String AUTH_TOKEN = "auth_token";
```
Comment (nuth) [inline serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/logging/SRMarkerFactory.java]: MINOR Move this variable to comply with Java Code Conventions. [rule](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1213)

**Comment by nuth** [comment]
SonarQube analysis reported 3 issues
* MINOR 3 minor

Watch the comments in this conversation to review them.

File: serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/logging/SRMarkerFactory.java (hunk 2 above)
Comment (martinwam) [inline serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/logging/SRMarkerFactory.java]: :-1:

**Pull request closed without merging**